### PR TITLE
CQ migration guide

### LINKED ISSUE #1803: Document migrating CQs to tasks
We need to document how to migrate 1.x CQs to 2.0 tasks. Here's a start for the content: 

1. Use the InfluxDB 1.x `influx` interactive shell and run `show continuous queries`:

    ```sh
    $ influx
    Connected to http://localhost:8086 version 1.8.3
    InfluxDB shell version: 1.8.3
    > show continuous queries
    ```

2. Save the continuous queries and convert them to Flux. For help converting InfluxQL to Flux, use the following resources:

    - [Get started with InfluxDB tasks](/influxdb/v2.0/process-data/get-started/)
    - [Get started with Flux](/influxdb/v2.0/query-data/get-started/)
    - [Common tasks](/v2.0/process-data/common-tasks/#downsample-data-with-influxdb)
    - [`influx transpile`](https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/transpile/) (transpiles InfluxQL into Flux)
    - Post in the [InfluxData Community](https://community.influxdata.com/)
    - Ask in the [InfluxDB Community Slack](https://influxdata.com/slack)

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1803 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: Migrate continuous queries from InfluxDB 1.x to 2.0
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:2]: Maybe "... to 2.0 tasks"?

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+description: >
+  Upgrade from InfluxDB 1.x to InfluxDB 2.0.
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:4]: Needs revised—

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+    {{< keep-url >}}
+    ```sh
+    $ influx
+    Connected to http://localhost:8086 version 1.8.3
+    InfluxDB shell version: 1.8.3
+    > show continuous queries
+    ```
+
+2. Copy and save all output continuous queries to your local machine.
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:34]: Wording feels a bit off.

Maybe "Copy and save the displayed..."

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+The InfluxQL to Flux transpiler is in active development and is considered experimental.
+Transpiled queries may not be optimized for performance and may not fully replicate
+the behavior of the provided InfluxQL query.
+{{% /note %}}
+
+#### Modify your continuous query to work with the transpiler
+To transpile a continuous query to Flux, you must modify the InfluxQL:
+
+1. Move the database and retention policy into the `FROM` clause.
+2. Remove the continuous query-specific statements and clauses (`CREATE CONTINUOUS QUERY`, `ON`, `BEGIN`, `END`)
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:82]: Period at end of line.

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+##### Modified to work with the influx transpile command
+```sql
+SELECT mean("example-field")
+FROM "my-db".."example-measurement"
+GROUP BY time(1h)
+```
+
+#### Transpile the modified query
+Use the `influx transpile` command to transpile your modified InfluxQL into Flux.
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:106]: "transpile", or "convert"?

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  GROUP BY time(1h)
+'
+```
+
+## Create new InfluxDB tasks
+After converting your continuous query to Flux, use the Flux query to
+[create a new task](/influxdb/v2.0/process-data/manage-tasks/create-task/).
+
+## Other helpful resources
+The following resources are also available to help convert InfluxQL queries to Flux.
```
Comment (pierwill) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:121]: "...helpful when converting..."

Comment (pierwill) [reply content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:121]: (Just a thought.)

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+queries into Flux tasks.
+
+{{% note %}}
+The InfluxQL to Flux transpiler is in active development and is considered experimental.
+Transpiled queries may not be optimized for performance and may not fully replicate
+the behavior of the provided InfluxQL query.
+{{% /note %}}
+
+#### Modify your continuous query to work with the transpiler
+To transpile a continuous query to Flux, you must modify the InfluxQL:
```
Comment (timhallinflux) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:79]: I don't think we should advertise the transpile command at all.
in fact, I feel like we should probably drop it.  It isn't being developed.

**Review by pierwill — APPROVED** [review]
A few nits. Looks good!

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+  |> filter(fn: (r) => r._measurement == "example-measurement")
+  |> filter(fn: (r) => r._field == "example-field")
+  |> aggregateWindow(every: 1h, fn: mean)
+  |> to(
+    org: "example-org",
+    bucket: "my-db/downsampled-daily"
+  )
+```
+
+### influx transpile command
```
Comment (timhallinflux) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:67]: remove....I think.

**Review by timhallinflux — CHANGES_REQUESTED** [review]

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+  SELECT mean("example-field")
+  INTO "average-example-measurement"
+  FROM "example-measurement"
+  GROUP BY time(1h)
+END
+```
+
+##### Equivalent Flux query
+```js
+from(bucket: "my-db/")
```
Comment (kelseiv) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:58]: forward slash intentional?

File: content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+  |> range(start: -1d)
+  |> filter(fn: (r) => r._measurement == "example-measurement")
+  |> filter(fn: (r) => r._field == "example-field")
+  |> aggregateWindow(every: 1h, fn: mean)
+  |> to(
+    org: "example-org",
+    bucket: "my-db/downsampled-daily"
```
Comment (kelseiv) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:65]: downsample-daily?

Comment (kelseiv) [inline content/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs.md:65]: Perhaps explicitly point out differences that may make folks wonder? Like why there isn't "average-example-measurement" in Flux query, and example of how time "range" was set for CQs.

**Pull request merged**